libopae/plugins/xfpga/sysfs.c

```diff
@@ -48,10 +49,267 @@
 #include "sysfs_int.h"
 #include "common_int.h"
 
+// substring that identifies a sysfs directory as the FME device.
+#define FPGA_SYSFS_FME "fme"
+#define FPGA_SYSFS_FME_LEN 3
+// substring that identifies a sysfs directory as the AFU device.
+#define FPGA_SYSFS_PORT "port"
+#define FPGA_SYSFS_PORT_LEN 4
+#define OPAE_KERNEL_DRIVERS 2
+static struct {
+	const char *sysfs_class_path;
+	const char *sysfs_region_fmt;
```

Comment:
Add Device or region path
to Access Device id & vendor id

Reply:
This is being done incrementally. Will do this in a future PR.